demo/admin/src/links/Link.tsx

```diff
@@ -55,6 +55,7 @@ export const Link: DocumentInterface<Pick<GQLLink, "content">, GQLLinkInput> & D
         return null;
     },
     menuIcon: LinkIcon,
+    hasNoPreview: true,
```

Comment:
I'd include site preview in the name. Also, should we remove the implied double negative (`hasNoPreview: false`)? For example, we could change it to `showSitePreviewButton` or `hasSitePreview`

Reply:
> Also, should we remove the implied double negative (hasNoPreview: false)? For example, we could change it to showSitePreviewButton or hasSitePreview

The problem here, in my opinion, is that I can't set a default value and `undefined` and `false` both evaluate to false. So you always have to  remember to do a strict type check because undefined means true

Reply:
Personally I'd be fine with doing a strict equality check, but I understand your argument.